rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-utility.yml

```diff
@@ -46,6 +46,22 @@
   when: "'xxx' in '{{ item.split(' ')[-1] }}'"
   with_items: neutron_output.stdout_lines|default([])
 
+- name: Grab relevant output of neutron port-list
```

Comment:
In `gather_stats.yml`, you used `source ~/openrc &&`, which is a little easier to read and understand. Could we use the same here?